spec/ttml1.xml

```diff
@@ -7469,27 +7469,25 @@ associated (owning) element node.</p>
 <gitem>
 <label>[associate region]</label>
 <def>
-<p>A <loc href="#element-vocab-type-content">Content</loc> element is associated with a region according
-to the following ordered rules, where the first rule satisfied
-is used and remaining rules are skipped:</p>
+<p>A <loc href="#element-vocab-type-content">Content</loc> element <emph>E</emph> is associated 
+with a region <emph>R</emph> if
+any of the following ordered assertions is true, where the first assertion satisfied is used and remaining assertions are skipped:</p>
```

Comment:
First, I'd use "conditions" instead of "assertions" everywhere on this line for clarity.

Second, as this is written, I don't understand what difference it makes to say "any ... is true" and follow it by "where the first is used and remaining ones are skipped". Surely this is simply the logical consequence of an OR relationship - once you've found one that evaluates to true then you know the answer is true and you don't have to evaluate the others.

Hence I think the "where the first assertion satisfied is used and remaining assertions are skipped" text is unnecessary and should be removed.

It would be necessary if the mechanism by which the element was associated with the region were used later, but I believe it is not.

**However**, see comments below - it may be that the intent is not being clearly captured, and there is a useful stopping mechanism that can return an early "false".